source/settings/read-only.txt

```diff
@@ -0,0 +1,111 @@
+.. _compass-read-only:
+
+====================================
+Restrict Write Operations to MongoDB
+====================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can use the ``readOnly`` option to prevent users from performing write 
+operations to your MongoDB deployment through |compass-short|. If you enable the 
+``readOnly`` option, users cannot modify documents, create indexes, or specify 
+validation rules.
+
+About This Task 
+---------------
+
+By default, |compass-short| disables the ``readOnly`` option.
+
+If the ``readOnly`` option is enabled, |compass-short| you cannot enable the 
```

Comment:
[typo]

I don't think we want "Compass" in this sentence?

Reply:
Oop sorry bout that! Fixed.